iseus/index.html

```diff
@@ -125,7 +125,76 @@ <h3>Lorem ipsum dolor sit</h3>
         <div>
             <h1 class="h1">News</h1>
             <hr class="hr">
+        </div>
+        <div class="grid-container">
+            <div id="news1">
```

Comment:
You don't give a tag Id unless you'd like to reference it later (e.g. in javascript, or special style). I see no point in giving each div a unique Id